website/docs/api/tok2vec.mdx

```diff
@@ -100,6 +100,39 @@ pipeline components are applied to the `Doc` in order. Both
 | `doc`       | The document to process. ~~Doc~~ |
 | **RETURNS** | The processed document. ~~Doc~~  |
 
+## Tok2Vec.distill {id="distill", tag="method,experimental", version="4"}
+
+Train a pipe (the student) on the predictions of another pipe (the teacher). The
+student is typically trained on the probability distribution of the teacher, but
+details may differ per pipe. The goal of distillation is to transfer knowledge
+from the teacher to the student.
```

Comment:
I think in this case the documentation should be different, since the student tok2vec doesn't learn anything from the teacher tok2vec.

Reply:
Fixed.